### PR TITLE
Replaced the underlying Cassandra image

### LINKED ISSUE #11: [Cassandra] Docker image
hi,
i would like to deply jaegertracing on a Kubernetes cluster using production YAML files, but i see that Cassandra image use is **jpkroehling/cassandra** and not the [official image](https://hub.docker.com/_/cassandra/).

It is possible to use the official image ?

### PR DESCRIPTION
**Pull request by jpkrohling**
Fixes #11

### CODE REVIEW

**Comment by jpkrohling** [comment]
Blocked by: uber/jaeger#278 . This won't work until the image `jaegertracing/jaeger-cassandra-schema` is published with those changes.

**Comment by pavolloffay** [comment]
@jpkrohling could you please add a test which would ping `/rest/dependencies` to the BaseTest? Currently dep. endpoint returns an error due to incompatible C* version and this PR fixes it. It resolves #16

**Comment by jpkrohling** [comment]
The test was there already (thanks!), I just needed to remove the override with the `@Ignore`. It's still failing because the create-schema image isn't available yet.

File: production/src/test/java/io/jaegertracing/kubernetes/ProductionETest.java [hunk 1]
```diff
@@ -14,15 +14,9 @@
...
 import io.jaegertracing.kubernetes.deployment.BaseETest;
-import java.io.IOException;
-import org.junit.Ignore;
 
 /**
  * @author Pavol Loffay
  */
 public class ProductionETest extends BaseETest {
-
-  @Ignore("dependency links returns 404 because of old Cassandra image")
```
Comment (pavolloffay) [inline production/src/test/java/io/jaegertracing/kubernetes/ProductionETest.java]: This test will still fail even with fixed c*.

I would like to see added a test which would ping dependencies REST endpoint.

**Comment by jpkrohling** [comment]
Tests are now passing locally:
```
[INFO] Reactor Summary:
[INFO] 
[INFO] jaegertracing-kubernetes-parent .................... SUCCESS [  1.468 s]
[INFO] jaegertracing-kubernetes-deployment-itest .......... SUCCESS [  2.478 s]
[INFO] jaegertracing-kubernetes-all-in-one ................ SUCCESS [ 18.992 s]
[INFO] jaegertracing-kubernetes-production ................ SUCCESS [02:47 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:10 min
[INFO] Finished at: 2017-07-21T10:24:00+02:00
[INFO] Final Memory: 30M/311M
```

**Comment by jpkrohling** [comment]
@pavolloffay I think this is now ready to be merged.

File: production/src/test/java/io/jaegertracing/kubernetes/ProductionETest.java [hunk 2]
```diff
@@ -14,15 +14,16 @@
 package io.jaegertracing.kubernetes;
```
Comment (pavolloffay) [inline production/src/test/java/io/jaegertracing/kubernetes/ProductionETest.java]: There shouldn't be change in this file

File: jaegertracing-kubernetes-deployment-itest/src/main/java/io/jaegertracing/kubernetes/deployment/BaseETest.java [hunk 3]
```diff
@@ -128,6 +134,14 @@ public void testDependencyLinks() throws IOException, InterruptedException {
     }
   }
 
+  @Test
+  public void hitDependencyScreen() throws IOException {
+    InputStream response = new URL(queryUrl + "/api/dependencies?endTs=0").openStream();
```
Comment (pavolloffay) [inline jaegertracing-kubernetes-deployment-itest/src/main/java/io/jaegertracing/kubernetes/deployment/BaseETest.java]: Could you use the same http client as in other tests? There is probably missing an assert on the status code.

**Pull request merged**